Formula/dvc.rb

```diff
@@ -6,6 +6,7 @@ class Dvc < Formula
   url "https://files.pythonhosted.org/packages/d2/f7/88b9578181fc14a812a95e9fc92cfa2da8e4e857a4b247af36bf85e0a3d8/dvc-2.9.3.tar.gz"
   sha256 "513d0d9ee3c3123e616f760eb8556ee569121286c3e400a1fe4b3bf66e6dea29"
   license "Apache-2.0"
+  revision 1
```

Comment:
This is a rev-bump only change. Is that intentional?

Reply:
Hah, I believe that it got updated between when I originally sent the PR and when I rebased.